x-pack/plugins/cases/public/containers/configure/use_get_case_configuration.test.tsx

```diff
@@ -24,90 +22,72 @@ describe('Use get case configuration hook', () => {
     jest.clearAllMocks();
   });
 
-  it('calls the api when invoked with the correct parameters', async () => {
+  it('returns a configuration matching the owner', async () => {
     const spy = jest.spyOn(api, 'getCaseConfigure');
+    const targetConfiguration = {
+      id: 'my-new-configuration-3',
+      owner: mockedTestProvidersOwner[0], // used in the AppMockRenderer
+    };
+    spy.mockResolvedValue([
+      // @ts-expect-error: no need to define all properties
```

Comment:
super nit: you can avoid using the `@ts-expect-error` by spreading the `initialConfiguration`.